migrations/08_test_medianizer.js

```diff
@@ -1,3 +1,4 @@
 module.exports = function(deployer) {
-  deployer.deploy(artifacts.require("TestMedianizer"));
+  if (artifacts.require("Medianizer").address == null)
```

Comment:
Data for whether a Medianizer exists for a network has been put into `networks.json`.